android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1543,16 +1546,13 @@ public static void getTxExtraInfo(BraveWalletBaseActivity activity, NetworkInfo
                 });
     }
 
-    public static void getP3ANetworks(Callbacks.Callback1<NetworkInfo[]> callback) {
-        BraveActivity activity = BraveActivity.getBraveActivity();
-        if (activity == null) {
-            callback.call(new NetworkInfo[0]);
-        }
+    public static void getP3ANetworks(WalletModel walletModel,
+            Callbacks.Callback1<NetworkInfo[]> callback) {
```

Comment:
We should try to minimise the dependency if possible, to make utility method clean and accessiable so instead of passing the wallet model pass the `mCryptoNetworks` list from fragment?

Reply:
Updated, the function now accept `NetworkInfo[]` as parameter.